tests/src/test/scala/actionContainers/PythonActionContainerTests.scala

```diff
@@ -212,25 +217,6 @@ class PythonActionContainerTests extends BasicActionRunnerTests with WskActorSys
         })
     }
 
-    it should "report error if zipped Python action has virtualenv created in wrong environment" in {
```

Comment:
back and changed to support python2 and python3 environments
testcase: try to run python2 virtualenv in python3 container and vice versa